tools/codesigningtool/codesigningtool.py

```diff
@@ -244,10 +245,20 @@ def generate_arg_parser():
       "--disable_timestamp", action="store_true", help="disables the use of "
       "timestamp services"
   )
+  parser.add_argument("extra", nargs = argparse.REMAINDER, help="additional "
+      "arguments that go directly to codesign, starting with a '--' argument")
```

Comment:
What's the need for the extra `--`? just a sanity check?

Reply:
according to argparse docs, "If you have positional arguments that must begin with - and don’t look like negative numbers, you can insert the pseudo-argument '--' which tells parse_args() that everything after that is a positional argument:"

Reply:
i also like that it mirrors `bazel run` and its use of `--` in a way